src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #fff;
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.searchBar {
+  display: block;
+  position: relative;
+}
+
+.searchBar__img--big {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.searchBar__img--small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.searchBar__input {
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.searchBar__input--big {
+  height: 70px;
+  padding-left: 62px;
+}
+
+.searchBar__input--small {
+  height: 42px;
+  padding-left: 33px;
+}
+
+.searchBar__input--big::placeholder {
+  color: #3d4e61;
+  font-size: 16px;
+  line-height: normal;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 400;
+}
+
+.searchBar__input--small::placeholder {
+  color: #3d4e61;
+  font-size: 14px;
+  line-height: normal;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 400;
+}
```

Comment:
move repeated styles to searchBar__input
add a correct value to line-height that match design